include/cantera/oneD/Domain1D.h

```diff
@@ -478,11 +549,26 @@ class Domain1D
         m_force_full_update = update;
     }
 
+	// added by udri to support plane equation
+
+	virtual void setFlameControl(bool strainRateEqEnabled, 
+								 bool unityLewisNumber,
```

Comment:
Something went wrong with your indentation here. Please use spaces, not tabs, to indent your code.

Reply:
Thanks for the comment I will look into it.

Reply:
I will get back to you about the University of Dayton Copyright statement next week.